### ISSUE TITLE
pfm_app_deprecated not appearing?

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#138]
In working on updating the Google Chrome manifest, found that in 0.2.5-build5 a preference that I had specified a `pfm_app_deprecated` string was not showing up as deprecated in the GUI.  Not sure if this is a bug or I'm just not aware of how PC handles determining whether or not to show a preference as deprecated.

**Comment by apizz** [comment]
Appearing in latest beta build

**Comment by apizz** [comment]
Stumbled on a pref for which deprecated is still not showing.

Pref: `DefaultMediaStreamSetting`

```
<dict>
	<key>pfm_app_deprecated</key>
	<string>25</string>
	<key>pfm_app_min</key>
	<string>22</string>
	<key>pfm_description</key>
	<string>Allows you to set whether websites are allowed to get access to media capture devices. Access to media capture devices can be allowed by default, or the user can be asked every time a website wants to get access to media capture devices. - Documentation doesn't indicate when this was deprecated, but AudioCaptureAllowed was added in M25, so making an assumption.</string>
	<key>pfm_description_reference</key>
	<string>Allows you to set whether websites are allowed to get access to media capture devices. Access to media capture devices can be allowed by default, or the user can be asked every time a website wants to get access to media capture devices.

If this policy is left not set, 'PromptOnAccess' will be used and the user will be able to change it.

2 = Do not allow any site to access the camera and microphone
3 = Ask every time a site wants to access the camera and/or microphone</string>
	<key>pfm_documentation_url</key>
	<string>https://dev.chromium.org/administrators/policy-list-3#DefaultMediaStreamSetting</string>
	<key>pfm_name</key>
	<string>DefaultMediaStreamSetting</string>
	<key>pfm_range_list</key>
	<array>
		<integer>2</integer>
		<integer>3</integer>
	</array>
	<key>pfm_range_list_titles</key>
	<array>
		<string>Do not allow any site to access the camera and microphone</string>
		<string>Ask every time a site wants to access the camera and/or microphone</string>
	</array>
	<key>pfm_title</key>
	<string>Default mediastream setting</string>
	<key>pfm_type</key>
	<string>integer</string>
</dict>
```
![Screen Shot 2019-03-31 at 11 35 16 PM](https://user-images.githubusercontent.com/11789931/55302107-ab7e4d00-540d-11e9-9f18-1b47b412db67.png)
![Screen Shot 2019-03-31 at 11 35 20 PM](https://user-images.githubusercontent.com/11789931/55302112-ade0a700-540d-11e9-8ed8-768aad7a15ed.png)

**Comment by erikberglund** [comment]
Fixed in the next version

**Comment by apizz** [comment]
Fixed in 0.2.5 release
